tex/gregoriotex-syllable.tex

```diff
@@ -1075,9 +1116,9 @@
         \gre@skip@temp@two=0 pt\relax%
       \fi %
       \gre@debugmsg{ifdim}{ previousenddifference < 0pt}%
-      \ifdim\gre@dimen@previousenddifference < 0 pt\relax%
+      \ifdim\gre@dimen@temp@three < 0 pt\relax%
         \advance\gre@skip@temp@two by %
```

Comment:
This change and all subsequent ones in this file are in the old algorithm, not the new one.  Are they really necessary or did you get a little over zealous with find-and-replace?